base_report_to_printer/wizards/printing_printer_update_wizard_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="printer_update_wizard" model="ir.ui.view">
+        <field name="name">printing.printer.update.wizard</field>
+        <field name="model">printing.printer.update.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Update Printers from CUPS">
+                <div name="introduction">
+                    <p>
+                    This process will create all missing printers from the current CUPS server."
+                    </p>
+                </div>
+                <footer>
+                    <button
+                        name="action_ok"
+                        string="Ok"
+                        type="object"
+                        class="btn-primary"
+                    />
+                    <button string="Cancel" class="btn-default" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <record id="action_printer_update_wizard" model="ir.actions.act_window">
+        <field name="name">Update Printers from CUPS</field>
+        <field name="res_model">printing.printer.update.wizard</field>
+        <field name="view_mode">form</field>
+        <field name="target">new</field>
+    </record>
+    <menuitem
+        action="action_printer_update_wizard"
+        sequence="40"
+        id="menu_printer_update_wizard"
```

Comment:
this menu (Update Printers from CUPS) is useless, no group has `create` right on this wizard, even Admin user will face this error:
![image](https://github.com/user-attachments/assets/f794a52c-0bec-4000-bd94-3218dceef732)

Reply:
Hi anh, I fixed and included in https://github.com/trisdoan/report-print-send/pull/1